synapse/lib/certdir.py

```diff
@@ -194,6 +204,36 @@ def genUserCert(self, name, signas=None, outp=None, pkey=None):
 
         return pkey, cert
 
+    def genClientCert(self, name, outp=None, signas=None):
```

Comment:
`signas` is not an accurate variable name, what we are passing in here is the name of the CA. We use this name to load the CA certificate to be packed into the PKCS12 file, we do not use its key and do not sign anything with it.